### ISSUE TITLE
Listing 10.5 - Column function return issue

### ISSUE THREAD

**Issue by pelemarse** [bitfield/tpg-tools#1]
https://github.com/bitfield/tpg-tools/blob/a0387e2bc60f86646f88b96f8e223224783153e5/10.5/pipeline.go#L57-L59

maybe it should be

```go
	p.Reader = result
	return p
```

**Comment by bitfield** [comment]
Yes, I think so, because otherwise we lose whatever the pipeline's `Output` was set to. Good spotting, thank you!

**Comment by bitfield** [comment]
Fixed in e284c67.
